src/js/components/Settings/VoterPhoneVerificationEntry.jsx

```diff
@@ -580,49 +580,45 @@ class VoterPhoneVerificationEntry extends Component {
             </FirstRowPhoneOrEmail>
             <SecondRowPhoneOrEmail>
               {isPrimarySMSPhoneNumber ? (
-                <div key={`${voterSMSPhoneNumberFromList.sms_we_vote_id}-internal`}>
+                <SecondRowPhoneOrEmailDiv>
```

Comment:
I think the key is still needed, for when we have multiple phone numbers on one account.